src/main/java/de/flapdoodle/embed/mongo/distribution/Version.java

```diff
@@ -223,7 +235,7 @@ public String toString() {
 
 		@Deprecated
 		LEGACY(V2_6),
-		PRODUCTION(V3_0),
+		PRODUCTION(V3_2),
```

Comment:
the idea behind this version stuff is:
- development should point to the newest dev release (currently: 3.2.1... )
- production should point to latest prod release (V3_2 is correct)
- V3_0 should point to the latests release for 3.0.x

so it would be better to keep V3_0(V3_0_5) for people using this "hook" for the desired version.